department-of-veterans-affairs/caseflow
# Added correspondence_type migration with seeds.

Resolves [Create Correspondence Type Lookup Database Table](https://jira.devops.va.gov/browse/APPEALS-29187)

# Description
Added a migration for correspondence_table and added seed data as well.

## Acceptance Criteria

## Testing Plan

1. Open a rails console
2. Execute the following commands
3. Run a migration to update the table changes.
```
require Rails.root.join("db/seeds/base.rb").to_s

Dir[Rails.root.join("db/seeds/*.rb")].sort.each { |f| require f }

Seeds::CorrespondenceTypes.new.seed!
```

4. You should now have the correct Correspondence_Type data in the table.
5. You can verify the correspondence_type data with `CorrespondenceType.all`

# Backend
## Database Changes
*Only for Schema Changes*

* [ X] Update column comments; include a "PII" prefix to indicate definite or potential [PII data content](https://github.com/department-of-veterans-affairs/appeals-team/blob/master/caseflow-team/0-how-we-work/pii-handbook.md#what-is-pii)


## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:
